app/server.rb

```diff
@@ -95,27 +96,39 @@ def ensureConnect(dbConfig)
 $s3Bucket = Aws::S3::Bucket.new($s3Config.bucket, client: $s3Client)
 
 # Internal modules to implement specific pages and functionality
-require_relative 'dbCache'
+require_relative '../util/sanitize.rb'
+require_relative '../util/xmlutil.rb'
+require_relative '../util/event.rb'
 require_relative 'hierarchy'
 require_relative 'listViews'
 require_relative 'searchApi'
 require_relative 'queueWithTimeout'
 require_relative 'unitPages'
 require_relative 'citation'
 require_relative 'loginApi'
-require_relative 'fileCache'
-require_relative '../util/sanitize.rb'
-require_relative '../util/xmlutil.rb'
-require_relative 'merritt'
+require_relative 'fetch'
+
+class StdoutLogger
+  def << (str)
```

Comment:
Huh?

Reply:
Are you asking about the syntax here? Basically this is a method that gets called when you do this:

$stdoutLogger << "stuff to log"

In my case I want to catch those occurrences and funnel them to my globally overridden "puts" which prepends the worker number and thread number to the output (and also serializes output with a mutex).

Did that answer the question?

Reply:
Yes thanks